sss/tests/test_sss.py

```diff
@@ -18,6 +22,25 @@ def test_read_file(setup_and_teardown_package):
 
     #session = setup_and_teardown_package.sessionmaker()
 
+def test_check_extra_columns(setup_and_teardown_package):
+    """ Test checking the extra columns: miscellaneous_is_secondary, health_care_is_secondary, analysis_is_secondary """
+
+    df, file = sss_table.read_file(os.path.join(DATA_PATH, 'AR2022_SSS_Full.xlsx'))
+    df, miscellaneous, health_care, arpa = sss_table.check_extra_columns(df)
+
+    assert 'miscellaneous_is_secondary' and 'health_care_is_secondary' and 'analysis_is_secondary' in df.columns
+
+def test_check_extra_columns_error(setup_and_teardown_package):
+    with pytest.raises(ValueError, match="df should be a pandas dataframe."):
+        df, miscellaneous, health_care, arpa = sss_table.check_extra_columns((os.path.join(DATA_PATH)), 'AR2022_SSS_Full.xlsx')
+
+def test_data_folder_to_database(setup_and_teardown_package):
+    db = AutomappedDB(test_db_url)
+    session = db.sessionmaker()
+    sss_table.data_folder_to_database(os.path.join(DATA_PATH, db_url=test_db_url))
+    result = session.query('SSS').filter('SSS'.State == 'CA').all()
```

Comment:
This doesn't work. You need to import the `SSS` object using `from sss import SSS`. Then here you pass that object not a string. Also, you need to specify a state that is in your test data. Maybe `"FL"`?